From 2ea8be10ba05260e4f86f9d3ad8dc822d501fca5 Mon Sep 17 00:00:00 2001
From: "Christopher A. Williamson" <contact@chrisaw.io>
Date: Tue, 29 Sep 2026 14:05:29 +0100
Subject: [PATCH 7/7] grant rpc servicemanager aidl visibility to the gsi stubs

the vendored virtualmachineservice aidl interface under the phh treble
gsi stubs imports the rpc servicemanager aidl interface, which is
otherwise restricted to the servicemanager and the virtualization tree.
---
 libs/binder/Android.bp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libs/binder/Android.bp b/libs/binder/Android.bp
index b3caaae..7c0ff5f 100644
--- a/libs/binder/Android.bp
+++ b/libs/binder/Android.bp
@@ -1018,6 +1018,7 @@ aidl_interface {
     },
     visibility: [
         ":__subpackages__",
+        "//device/phh/treble/gsi_stubs:__subpackages__",
         "//system/tools/aidl:__subpackages__",
         "//frameworks/native/cmds/servicemanager/rpc_servicemanager",
         "//packages/modules/Virtualization:__subpackages__",
-- 
2.55.0

